knimin/lib/data_access.py

```diff
@@ -2421,14 +2421,14 @@ def _study_exists(self, study_id):
         if not self._con.execute_fetchone(sql, [study_id])[0]:
             raise ValueError('Study ID %s does not exist.' % study_id)
 
-    def _study_is_unique(self, qiita_study_id=None, title=None, study_id=None):
+    def _study_is_unique(self, qiita_study_id=None, title=None, skip_id=None):
         """Confirms that a Qiita study ID and/or a title is not duplicate
 
         Parameters
         ----------
         qiita_study_id : int
         title : str
-        study_id : int
+        skip_id : int
```

Comment:
This "skip_id" is for the "edit_study" function but not for the "create_study" function. When the user attempts to rename a study, and types a name that is identical to another study (defined by "skip_id"), an error will be raised. I will add more docstring to explain this.